awx/ui_next/src/components/AppContainer/AppContainer.jsx

```diff
@@ -0,0 +1,217 @@
+import React, { Component, Fragment } from 'react';
+import { withRouter } from 'react-router-dom';
+import { global_breakpoint_md } from '@patternfly/react-tokens';
+import {
+  Nav,
+  NavList,
+  Page,
+  PageHeader as PFPageHeader,
+  PageSidebar,
+} from '@patternfly/react-core';
+import styled from 'styled-components';
+import { t } from '@lingui/macro';
+import { withI18n } from '@lingui/react';
+
+import { ConfigAPI, MeAPI, RootAPI } from '../../api';
+import About from '../About';
+import AlertModal from '../AlertModal';
+import ErrorDetail from '../ErrorDetail';
+import { ConfigProvider } from '../../contexts/Config';
+
+import NavExpandableGroup from './NavExpandableGroup';
+import BrandLogo from './BrandLogo';
+import PageHeaderToolbar from './PageHeaderToolbar';
+
+const PageHeader = styled(PFPageHeader)`
+  & .pf-c-page__header-brand-link {
+    color: inherit;
+
+    &:hover {
+      color: inherit;
+    }
+
+    & svg {
+      height: 76px;
+    }
+  }
+`;
+
+class AppContainer extends Component {
```

Comment:
I'm converting this to a functional component but haven't pushed the changes to this PR yet.